factorio

```diff
@@ -98,23 +113,23 @@ refresh_save(){
   # Else we copy the latest save to our own save file
   debug "using refreshed save"
   if ! as_user "cp ${lastsave} ${savedir}/${SAVE_NAME}.zip"; then
-    echo "Error! Unable to copy save"
-    return 1
+    echo "Error! Failed to refresh saves"
+    exit 1
   fi
 }
 
 find_pid(){
-  pid=$(ps ax | grep -v grep | grep "$(echo -e "${INVOCATION}" | sed -e 's/[[:space:]]*$//')" | grep -v " SCREEN " | awk '{ print $1}')
```

Comment:
It looks like you want to revert my latest fix for the find_pid here, is this an error on your part or do you have a specific reason?